app/models/manageiq/providers/openstack/manager_mixin.rb

```diff
@@ -133,7 +133,7 @@ def translate_exception(err)
       when Excon::Errors::NotFound
         MiqException::MiqHostError.new("Endpoint not found.")
       when Excon::Errors::Unauthorized
-        MiqException::MiqInvalidCredentialsError.new("Login failed due to a bad username or password.")
```

Comment:
I considered "due to a bad domain_id, username, or password" but felt like we might just be chasing an exhaustive list

Reply:
can we tell if it is because of an invalid domain_id vs userid/password?

Reply:
No all we get is a 401 unauthorized exception